modules/lib/test/IdProviderController.test-d.ts

```diff
@@ -12,9 +12,6 @@ const log = {
 const myIdProviderController = {
```

Comment:
Here you need to provide correct type for `const myIdProviderController` (`const myIdProviderController: IdProviderController`), then you will immediately see that type of `autoLogin` cannot be just `void`.